contracts/RemovalQueue.sol

```diff
@@ -188,4 +190,23 @@ library RemovalQueue {
     }
     return totalBalance;
   }
+
+  function fill(address val, uint256 length)
+    internal
+    pure
+    returns (address[] memory)
+  {
+    // uint256 n = from.length;
+    // address[] memory to = new address[](from.length);
+    assembly {
+      // Create an dynamic sized array manually.
+      let memOffset := mload(0x40) // 0x40 is the address where next free memory slot is stored in Solidity.
+      mstore(memOffset, 0x20) // single dimensional array, data offset is 0x20
```

Comment:
wondering if there's any reason to have consistency between hex and decimal for this 32 value

Reply:
oh, thought I caught that already and changed everything to decimal-- good catch